### PR TITLE
Module 3 : React Tutorial -  Todo app

### PR DESCRIPTION
**Pull request by ikennarichard**
## Module 3 (Week 2): React Tutorial - Todo App
For this project, I
- Used the React tutorial to develop the Todo App
- Used state to manage data flows between the components

### CODE REVIEW

**Review by zunairkhan811 — CHANGES_REQUESTED** [review]
## Hi @ikennarichard  🙋‍♂️
Good job working on your first Todo list React App. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement a few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Gitflow is used in a professional way ✅
- Good commits are used ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required Changes :recycle: 
- Kindly consider the inline review comments.



## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------


HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Comment by zunairkhan811** [comment]
- [ ] Please Kindly fix the name of your project in the Readme file as your project name is 'todo list' not math magician as this will have improve professionalism
readmeerror

File: README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math Magicians <a name="about-project"></a>
```
Comment (zunairkhan811) [inline README.md]: - [ ] Please kindly fix your Readme issue as the name of your project Is 'Todo list' not 'math-magician' which will increase professionalism. 😉

**Review by zunairkhan811 — APPROVED** [review]
- [x] Please kindly install linters in your project as linters are very helpful in making your project Error-free 😉

**Review by zunairkhan811 — CHANGES_REQUESTED** [review]
- [ ] Please kindly install linter files as linters are very helpful in making your project error free. 😉

File: README.md (hunk 1 above)
Comment (ikennarichard) [reply README.md]: Thanks a lot @zunairkhan811 for the corrections, it was really helpful for me. I've fixed the README issues and added the linters. Thanks again and have a great day!

**Review by zunairkhan811 — APPROVED** [review]
# Status: :green_circle:  Approved! :partying_face:
Hi @ikennarichard  👋, 

Your project is complete! Great job for nailing it :1st_place_medal:. There is nothing else to say other than... it's time to merge it  :shipit:  Congratulations! :tada::100::star2:

giphy



## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by zunairkhan811 — APPROVED** [review]
# Status: :green_circle:  Approved! :partying_face:
Hi @ikennarichard  👋, 

Your project is complete! Great job for nailing it :1st_place_medal:. There is nothing else to say other than... it's time to merge it  :shipit:  Congratulations! :tada::100::star2:

giphy



## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by zunairkhan811 — APPROVED** [review]
# Status: :green_circle:  Approved! :partying_face:
Hi @ikennarichard  👋, 

Your project is complete! Great job for nailing it :1st_place_medal:. There is nothing else to say other than... it's time to merge it  :shipit:  Congratulations! :tada::100::star2:

giphy



## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**